cypress/e2e/homeworks/hw-eshop-formular/automation_test_store_register.cy.js

```diff
@@ -0,0 +1,50 @@
+import { faker } from "@faker-js/faker";
+import { HomePage } from "../../../page-objects/automationteststore/home_page";
+import { CreateAccount } from "../../../page-objects/automationteststore/create_account_page";
+
+describe("Fill Automation Test Store Formular", () => {
+  it("Open eshop and register new account", () => {
+    const firstName = faker.person.firstName();
+    const lastName = faker.person.lastName();
+    const email = faker.internet.email({
```

Comment:
V zadání bylo pracovat s exampleEmailem, je to bezpečnější z toho pohledu, že se omylem netrefíš do reálného e-mailu